packages/ckeditor5-widget/src/widget.ts

```diff
@@ -247,6 +244,21 @@ export default class Widget extends Plugin {
 		const modelElement = editor.editing.mapper.toModelElement( element );
 
 		this._setSelectionOverElement( modelElement! );
+
+		function setSelectionInAncestorElement() {
```

Comment:
Why this is a nested function? I'd suggest extracting to a private method or a helper function.